### PR TITLE
documentation(EJ2-64046): searching using multiple keywords.

### PR DESCRIPTION
**Pull request by anbuarjun**

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/searching.md [hunk 1]
```diff
@@ -201,3 +201,35 @@ You can search the Grid data on each key stroke by binding the `keyup` event for
 {% endtabs %}
 {% endif %}
 
+
+
+## Perform search operation in Grid using multiple keywords
+
+You can perform a searching operation in the Grid using multiple keywords. This can be achieved by the [actionBegin](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_ActionBegin) event of the Grid.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/searching.md:208]: property name must be in caps lock started ActionBegin for core and mvc change it

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/searching.md [hunk 2]
```diff
@@ -202,6 +202,36 @@ You can search the Grid data on each key stroke by binding the `keyup` event for
...
+ {% include code-snippet/grid/search/search-multiple-keywords/razor %}
+{% endhighlight %}
+{% highlight c# tabtitle="Search-multiple-keywords.cs" %}
+{% include code-snippet/grid/search/search-multiple-keywords/search-multiple-keywords.cs %}
+{% endhighlight %}
+{% endtabs %}
+{% endif %}
+
+
+> * To search for multiple keywords, it only works if you use a comma separator to separate a values in search bar.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/searching.md:234]: Define multiple keywords by using a comma separator in search bar to search.

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/searching.md (hunk 1 above)
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/searching.md:208]: I will change that mam.

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/searching.md (hunk 2 above)
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/searching.md:234]: I will change that.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**